.github/workflows/helm-deploy.yaml

```diff
@@ -19,6 +19,11 @@ on:
         description: "Target environment for deployment (stg|prod)"
         required: true
         type: string
+      helm-wait:
```

Comment:
maybe this workflow event variable should be `atomic` (defaulting to false).
Then i would do the same thing except replace the WAIT_ARG if 
```
if [[ "${{ inputs.atomic }}" == true ]]; then
```


Or are there other options for helm that we might want to add in the future?

Reply:
Good call. I hesitated on the input to shell logic translation, but let's see if it works, or maybe you know it does. :-D